features/admin.claims.v1/components/edit/local-claim/edit-mapped-attributes-local-claims.tsx

```diff
@@ -92,6 +100,10 @@ export const EditMappedAttributesLocalClaims: FunctionComponent<EditMappedAttrib
 
     const { t } = useTranslation();
 
+    const readOnlyUserStoreSet: Set<string> = useMemo(() =>
+        new Set(readOnlyUserStoreNames?.map((name: string) => name.toUpperCase()) ?? [])
+    , [ readOnlyUserStoreNames ]);
```

Comment:
Any specific reason to create a Set from an array? IMO we can use the `readOnlyUserStoreNames` array directly.